powershell/Tests/lib/Start-Engine.ps1

```diff
@@ -57,8 +57,19 @@ function Invoke-Test {
 
 # Record the original environment variables.
 $originalEnv = @{ }
-foreach ($envVar in (Get-ChildItem -LiteralPath env:)) {
-    $originalEnv[$envVar.Name] = $envVar.Value
+foreach ($key in ([Environment]::GetEnvironmentVariables()).Keys) {
```

Comment:
this was a really bizarre problem. i didnt even know this was possible on windows.

Reply:
after i ran into this problem, i switched this script over to use the .net method everywhere for accessing env vars instead of the ps way.